Homework_7_Java_Operators_Conditions/JavaOperatorsAndConditions/src/MagicalApple.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class MagicalApple {
+
+    public static void main (String[] args) {
+
+
+        Scanner scanner = new Scanner(System.in); {

```

Comment:
This curly bracket doesn't break anything, but it's redundant, you can remove it with its pair and it will still work :)